internal/formatter.go

```diff
@@ -1,4 +1,4 @@
-package senders
+package internal
```

Comment:
There was a request to make these public earlier: https://github.com/wavefrontHQ/wavefront-sdk-go/issues/7

Moving it to a different package breaks backwards compatibility. We should consider deprecating existing code and leveraging Go [aliases](https://medium.com/a-journey-with-go/go-aliases-simple-and-efficient-8506d93b079e) where possible to smoothen the transition.